### PR TITLE
Twig prepared set

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Rector **upgrades your application** for you, with focus on open-source projects
     <a href="/src/config/level/phpunit"><img src="/docs/images/phpunit.jpg"></a>
     <img src="/docs/images/space.png">
     <a href="/src/config/level/roave"><img src="/docs/images/roave.png"></a>
+    <img src="/docs/images/space.png">
+    <a href="/src/config/level/twig"><img src="/docs/images/twig.png"></a>
```
Comment (greg0ire) [inline README.md]: Hmm... the logo is big...

Comment (TomasVotruba) [reply README.md]: I'll handle it, no woriies

**Comment by TomasVotruba** [comment]
Could you please rebase on current master?

Those fails should have been already resolved

**Comment by greg0ire** [comment]
Sure

**Comment by TomasVotruba** [comment]
Awesome work, thank you :clap: 

Just one detail, the config name should be *target version* to upgrade to from one right bellow it.
In this case `twig20.yml`. I'll handle it

**Comment by TomasVotruba** [comment]
Just one question, I've dig throught Twig CHANGELOG and it looks this change was done in:  1.34.0 and 2.4.0 too: https://github.com/twigphp/Twig/blob/2.x/CHANGELOG#L27

Are they compatible?
If so, maybe we should add `twig24.yml` and just include this config. What do you think?

**Comment by greg0ire** [comment]
I though since 1.34 < 2.4, this was how to deal with it, but maybe this reasoning is flawed. If people use 2, they will not consider using files starting with 1?

**Comment by TomasVotruba** [comment]
Well according to change log, version 2.3.x didn't have namespaces yet. Adding namespace was probably backported to 1.3 branch.

So if people use 2.3 and move to 2.4, they will use this config too

Could you confirm @nicolas-grekas?

**Comment by nicolas-grekas** [comment]
I don't remember the precise version number, but namespaces appeard at the same time in v1 and v2.
In a BC/optional way.

**Comment by greg0ire** [comment]
Maybe use a symlink?

**Comment by greg0ire** [comment]
Or maybe using version numbers in the filename is not right? Maybe describe the migration instead?

**Comment by TomasVotruba** [comment]
@nicolas-grekas Thanks, I assumed so.

@greg0ire Symlink would make sense only if  there were no other differences between version.
Using `import` includes single file and allows further extending the config.

**Comment by TomasVotruba** [comment]
Resolved: https://github.com/rectorphp/rector/commit/69105ffcb2487957d0affd0b4bc6c92c4ec55ab1

**Pull request merged**